### PR TITLE
Move all _ prefixed properties to Symbols

### PR DESCRIPTION
**Pull request by HugoLiconV**
I moved _prefixed properties in Reply.js to symbols.
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
I agree with @cemremengu! Good work!

**Comment by HugoLiconV** [comment]
Thanks, I'm fixing that

File: lib/symbols.js [hunk 1]
```diff
@@ -13,7 +13,12 @@ const keys = {
   kMiddlewares: Symbol('fastify.middlewares'),
   kCanSetNotFoundHandler: Symbol('fastify.canSetNotFoundHandler'),
   kFourOhFourLevelInstance: Symbol('fastify.404LogLevelInstance'),
-  kFourOhFourContext: Symbol('fastify.404ContextKey')
+  kFourOhFourContext: Symbol('fastify.404ContextKey'),
+  kReplySerializer: Symbol('reply.Serializer'),
+  kReplyIsError: Symbol('reply.isError'),
+  kReplyHeaders: Symbol('reply.headers'),
+  kReplyHasStatusCode: Symbol('reply.hasStatusCode'),
+  kReplyStartTime: Symbol('reply.errorHandlerCalled')
```
Comment (cemremengu) [inline lib/symbols.js]: oops things got mixed up here

File: lib/reply.js [hunk 2]
```diff
@@ -7,7 +7,13 @@ const statusCodes = require('http').STATUS_CODES
 const flatstr = require('flatstr')
 const FJS = require('fast-json-stringify')
 const {
-  kFourOhFourContext
+  kFourOhFourContext,
+  kErrorHandlerCalled,
```
Comment (cemremengu) [inline lib/reply.js]: This is left out, need a key with `Reply` prefix

File: lib/symbols.js [hunk 3]
```diff
@@ -13,7 +13,13 @@ const keys = {
   kMiddlewares: Symbol('fastify.middlewares'),
   kCanSetNotFoundHandler: Symbol('fastify.canSetNotFoundHandler'),
   kFourOhFourLevelInstance: Symbol('fastify.404LogLevelInstance'),
-  kFourOhFourContext: Symbol('fastify.404ContextKey')
+  kFourOhFourContext: Symbol('fastify.404ContextKey'),
+  kReplySerializer: Symbol('reply.Serializer'),
```
Comment (cemremengu) [inline lib/symbols.js]: `reply.Serializer` -> `reply.serializer` final nit from me then LGTM :) apologies for being a pain with my partial reviews

Comment (HugoLiconV) [reply lib/symbols.js]: Don't worry! jaja

File: lib/symbols.js [hunk 4]
```diff
@@ -13,7 +13,13 @@ const keys = {
   kMiddlewares: Symbol('fastify.middlewares'),
   kCanSetNotFoundHandler: Symbol('fastify.canSetNotFoundHandler'),
   kFourOhFourLevelInstance: Symbol('fastify.404LogLevelInstance'),
-  kFourOhFourContext: Symbol('fastify.404ContextKey')
+  kFourOhFourContext: Symbol('fastify.404ContextKey'),
+  kReplySerializer: Symbol('reply.serializer'),
```
Comment (delvedor) [inline lib/symbols.js]: I think the Symbol name should always be prefixed with `fastify`.
Eg: `fastify.reply.serializer`

@mcollina what do you think?

File: lib/symbols.js (hunk 3 above)
Comment (HugoLiconV) [reply lib/symbols.js]: @cemremengu Could you check it again? I already fixed it

**Comment by HugoLiconV** [comment]
@delvedor I already added the fastify prefix. Could you check it, please?

**Pull request merged**